kernels/etherlink/bench/src/results.rs

```diff
@@ -215,30 +214,33 @@ struct LogLine {
 struct TxLog {
     #[expect(unused)]
     address: String,
-    #[expect(unused)]
-    topics: Vec<String>,
     data: Vec<u8>,
 }
 
 #[derive(Debug)]
 enum TxExecutionResult {
-    CallSucceeded {
+    Success {
         #[expect(unused)]
         reason: String,
-        value: Option<u64>,
+        #[expect(unused)]
+        gas_used: u64,
```

Comment:
is it worth parsing the gas used/refunded etc if we never look at them?

Reply:
Probably not, removed [here](https://github.com/tezos/riscv-pvm/compare/bb3b75927c163db894f11fa564740c02ca971e99..5d41d1a492607462134dc5ec68e159ff2c6ff150).